core/tests/performance_tests/base.py

```diff
@@ -28,6 +30,21 @@ class TestBase(unittest.TestCase):
 
     BASE_URL = 'http://localhost:9501'
 
+    LIBRARY_URL_SUFFIX = '/library'
+    EDITOR_URL_SLICE = '/create/'
+    PLAYER_URL_SLICE = '/explore/'
+    PROFILE_URL_SLICE = '/profile/'
+    LOGIN_URL_SUFFIX = '/_ah/login'
+    ADMIN_URL_SUFFIX = '/admin'
+    CONTINUE_URL_SUFFIX = (
+        '?continue=http%3A//localhost%3A9501/signup%3Freturn_url%3D%252F')
+    DASHBOARD_URL_SUFFIX = '/dashboard'
+    SPLASH_URL_SUFFIX = '/splash'
+
+    random_id = random.randint(1, 100000)
+    EMAIL = 'test%d@example.com' % random_id
```

Comment:
Since, if a server is running and some test creates a user, then the flow changes if we use that user and our browser interactions won't work as intended. Hence, a new user every time.